### PR TITLE
fix(deps): update atom-package-deps

### PR DESCRIPTION
**Pull request by UziTech**
updating eslint will take some work but I figured we might as well update the other dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -180,19 +180,22 @@
...
+    "@commitlint/travis-cli": "12.0.1",
     "@semantic-release/apm-config": "8.0.0",
     "eslint-config-airbnb-base": "13.2.0",
     "eslint-plugin-import": "2.20.1",
-    "husky": "4.2.3",
+    "husky": "4.3.8",
     "jasmine-fix": "1.3.1",
     "rimraf": "3.0.2",
-    "semantic-release": "17.0.4"
+    "semantic-release": "17.4.2"
```
Comment (aminya) [inline package.json:192]: Sorry, for being late, but I wanted to remove these because our CI already supports linting the commits and semantic-release

**Review by aminya — CHANGES_REQUESTED** [review]
Could you rebase this so it only updates atom-package-deps? 

#1411 covers the rest

**Comment by UziTech** [comment]
done :+1:

File: spec/fixtures/import-resolution/.eslintrc.yaml [hunk 2]
```diff
@@ -9,4 +9,5 @@
     import/no-unresolved: 2
 
   parserOptions:
+    ecmaVersion: 2015
```
Comment (aminya) [inline spec/fixtures/import-resolution/.eslintrc.yaml:12]: I think this is not needed for this PR. It is covered in #1360

```suggestion
```

**Review by aminya — APPROVED** [review]

**Pull request merged**